docs/en/observability/uptime-set-up-heartbeat.asciidoc

```diff
@@ -5,6 +5,31 @@
 check the status of your services and determine if they are available. It gathers performance data,
 formats it, and sends the data to the {stack}.
 
+++++
+<div class="imageblock">
+<img
+  src="images/uptime-set-up-heartbeat.png"
+  class="w-100"
+  alt="Diagram showing which pieces of software are used to configure monitors, create monitors, and view results when using Heartbeat. Described in detail in Diagram text description."
+  aria-details="diagram-text-heartbeat">
+</div>
+++++
+
+[[diagram-text-heartbeat]]
+[%collapsible]
+.Diagram text description
+====
+. Configure monitors using the **{heartbeat} YAML file**. If configuring a _browser_ monitor, include either:
+  * **Inline journey code**
+  * **Reference to a zip URL** (If using a zip URL, you need a **Zipped project using `@elastic/synthetics`**.)
+. Optionally, use the **{heartbeat} Docker image** to install {heartbeat}.
+. When creating monitors, use either one of the following methods of **Authentication**:
+  * **{ecloud} ID**
+  * **Host, username, and password**
+.  Once authenticated, monitors push data to **{es}**.
```

Comment:
I think this will be confusing for users because we're tying the push command authentication to the authentication that a monitor must do to insert data into ES, which are different things.

I'd probably remove this sentence altogether.

Reply:
I don't want to remove the whole sentence, because we need to explain how the "Elasticsearch" box fits into the diagram. 

```suggestion
.  Monitors send data to **{es}**.
```